mate-scripts/configs/git-hooks/pre-commit

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
```

Comment:
no more husky, only hardcore 🙂

Reply:
Just to avoid additional configs inside homework repos